dips/dip-10.md

```diff
@@ -73,6 +75,32 @@ struct DiemIdDomain {
 * The Diem ID domain can be created as part of creating a `ParentVASP` account.
 * An entity (potentially the Association) may at some point choose to expose an open source application(*Indexer*) that will allow indexing available Diem IDs and could provide a convenient REST API for applications to fetch information about the domains. The API of such an indexer is out of scope of this RFC.
 
+## Diem ID domain events
+
+Blockchain contract that manages Diem ID domains must emit an event every time Diem ID domain is created, removed or updated. Those events are critical for applications to be able to efficiently index existing Diem ID domains.
+
+While Diem ID domains published into VASP accounts, Diem ID domain events are published under the Treasury Compliance account. We consolidate events under single account to allow indexers to follow single event stream.
```

Comment:
Diem ID domains *are* published into VASP accounts

Reply:
Do we explicitly state what this address is anywhere? I'm curious how folks would find it.

Reply:
Yes, this document mentions specific address (I can repeat it here too, but it feels redundand)